go/service/saltpack.go

```diff
@@ -359,11 +359,13 @@ func (h *SaltpackHandler) saltpackEncryptFile(ctx context.Context, arg keybase1.
 	}, nil
 }
 
-func (h *SaltpackHandler) saltpackEncryptDirectory(ctx context.Context, arg keybase1.SaltpackEncryptFileArg) (keybase1.SaltpackEncryptFileResult, error) {
+func (h *SaltpackHandler) saltpackEncryptDirectory(ctx context.Context, arg keybase1.SaltpackEncryptFileArg) (res keybase1.SaltpackEncryptFileResult, err error) {
+	defer h.G().NotifyRouter.HandleSaltpackOperationDone(ctx, keybase1.SaltpackOperationType_ENCRYPT, arg.Filename)
+
 	h.G().Log.Debug("encrypting directory")
 
 	if filepath.Clean(arg.Filename) == filepath.Clean(arg.DestinationDir) {
-		return keybase1.SaltpackEncryptFileResult{}, errors.New("source and destination directories cannot be the same")
+		return res, errors.New("source and destination directories cannot be the same")
```

Comment:
i prefer it the way it was so that we know that `res` is empty.  can you change it back here and elsewhere?